hack/build-registry-bundle.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+set -e
```

Comment:
Do we still want this in the final commit?

Reply:
well, yes. We do want it, but I probably didn't put it in a commit that makes sense for the change.

The issue I hit was 'make ocs-registry' would "pass" even if an error occurred during the docker build process. Without 'set -e', the exit code of `make ocs-registry` doesn't indicate anything useful. 

I'll break it into a separate commit at least.

Reply:
fixed. i broke this into a separate commit.